proto/storage/product_usage.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package storage;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "storage";
+
+// SecuredUnits represents a record of an aggregated secured clusters usage
+// metrics. The metrics are aggregated periodically, and put into the database.
+message SecuredUnits {
+    // id is not used to retrieve data, but serves mostly for compatibility with
+    // the current implementation of the query generator.
+    string id                           = 1 [(gogoproto.moretags) = 'sql:"pk,type(uuid)"'];
+    // timestamp stores the moment at which the values of the metrics below are
+    // aggregated.
+    google.protobuf.Timestamp timestamp = 2;
+    // num_nodes is the maximum number of secured nodes, observed across all
+    // registered clusters during last aggregation period.
+    int64 num_nodes                     = 3;
```

Comment:
I could see aggregations or the desire to query Max and Min of these 2 fields.  I think I would recommend making them searchable columns.  Most the necessary work to do that should be from the generated code.